gsi.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+  <include name="base-q.xml"/>
+  <remote name="phh" fetch="https://github.com/phhusson/" />
+  <remote name="b2g-gsi" fetch="https://github.com/b2g-gsi/"/>
+  
+  <!-- MOD B2G -->
+  <remove-project name="platform_build"/>
+  <remove-project name="platform_system_vold"/>
+  <remove-project name="platform_system_core"/>
+  <!--   <remove-project name="platform_system_sepolicy"/> -->
+  <project groups="kai" name="platform_build" path="build/make" remote="b2g-gsi" revision="gsi_gonk_android-10.0.0_r11">
```

Comment:
this is where we would need a fork and a PR for your branch, how can we review without? (same for all `gsi_gonk_android-10.0.0_r11`)